### DISCUSSION TITLE
Cloud GPU machine on Vast.ai

### DISCUSSION THREAD

**Discussion by gtoscano69** [Acly/krita-ai-diffusion#549] [General]
I was able to setup a Cloud GPU machine on Vast.ai
the connection working but when I run "generate"
this red text appears
Error: Error transferring http://154.20.254.95:50616/prompt - server replied: Not Allowed [url=http://154.20.254.95:50616/prompt, code=2021

**Comment by Kobayashimaru82** [comment]
You might have to manually upload the updated ComfyUi-IPAdapter files into the Vast.ai Jupyter ComfyUI/custom nodes/ComfyUI_IPAdapter_plus/ And then reboot the GPU.

Also with using a Vast.ai GPU, none of the selection tools work such as Remove Content, even after the 1.16.1 update. Although they do work on the Interstice Cloud.

**Reply by gtoscano69** [reply]
Hi, I use all the credit in Interstice Cloud…
I never use Jupyter…

**Comment by Acly** [comment]
Latest template for vast.ai works with plugin up to 1.15.0
Still need to upload a new image for the 1.16.x changes

> Error transferring http://154.20.254.95:50616/prompt - server replied: Not Allowed 

not sure where this error is coming from though, I might have time to try it tomorrow

**Reply by Kobayashimaru82** [reply]
I got that same error today. Solved it by switching GPU instances on Vast.ai.

Still not able to do any Inpainting or anything involving the selection tools. Is that because of the outdated template?

**Reply by Kobayashimaru82** [reply]
And now I'm getting a "Server execution error: Error while deserializing header: MetadataIncompleteBuffer" Also solved by switching Vast.ai GPU instances.

**Comment by Acly** [comment]
Updated image to 1.16.1 - should work with all features now
